src/diffusers/pipelines/t2i_adapter/pipeline_stable_diffusion_adapter.py

```diff
@@ -215,6 +217,33 @@ def __init__(
     ):
         super().__init__()
 
+        if hasattr(scheduler.config, "steps_offset") and scheduler.config.steps_offset != 1:
```

Comment:
this pipeline use the SD checkpoints so do the same deprecation as SD pipeline 
this is the reason the from_pipe test failed here https://github.com/huggingface/diffusers/actions/runs/8514833413/job/23321268337#step:9:73

i.e. when we do `pipe.from_pretrained()`, the deprecated scheduler config (clip_sample, step_offset) is not updated;  but when we do `pipe = ...from_pipe(pipe_sd)`, these config got updated when creating `pipe_sd`